roles/confluent.variables/defaults/main.yml

```diff
@@ -316,6 +326,9 @@ schema_registry_ssl_enabled: "{{ssl_enabled}}"
 ### Boolean to enable mTLS Authentication on Schema Registry
 schema_registry_ssl_mutual_auth_enabled: "{{ ssl_mutual_auth_enabled }}"
 
+### Only use to customize directory Schema Registry writes log files to. Default location is /var/log/confluent/schema-registry. NOTE- schema_registry.appender_log_path is deprecated
```

Comment:
Modify this variable to customize the directory that the Schema Registry writes log files to. Default location is /var/log/confluent/schema-registry. NOTE- schema_registry.appender_log_path is deprecated.